model/src/main/proto/arguments.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// The fragment arguments passed to instances of ExitProfileDialogFragment.
+message ExitProfileDialogArguments {
+  // Decides the correct menu item to be highlighted after canceling the ExitProfile dialog.
+  HighlightItem highlight_item = 1;
+}
+
+enum HighlightItem {
```

Comment:
Add a single line explanation for this too.

Reply:
Please suggest one.

Reply:
Added an explanation comment, please mention it again if requires nit changes.